src/tiled/propertybrowser.cpp

```diff
@@ -551,16 +552,13 @@ void PropertyBrowser::propertiesChanged(Object *object)
         updateCustomProperties();
 }
 
-void PropertyBrowser::componentAdded(Object *, const QString &)
+void PropertyBrowser::componentAdded(QList<Object *> objects, const QString &)
```

Comment:
Please always remember to use `const &` when passing implicitly shared containers around. Passing them by value generally means it'll need to increment and decrement the reference count, and risks causing detach when a non-const override is used.